src/Native/Windows/build-native.cmd

```diff
@@ -111,6 +111,13 @@ echo See: https://github.com/dotnet/coreclr/blob/master/Documentation/project-do
 exit /b 1
 
 :GenVSSolution
+:: Generate Version file
+if not exist "%__IntermediatesDir%\clrcompression" md "%__IntermediatesDir%\clrcompression"
```

Comment:
Yes. I actually don't need this anymore because now I'm dropping the file into bin/obj instead so that it can be referenced by any number of native projects, but I will log an issue to make sure that the dir exists in buildtools.